content/applications/hr/attendances/kiosks.rst

```diff
@@ -0,0 +1,191 @@
+======
+Kiosks
+======
+
+Odoo's *Attendances* app allows employees to check in and out of work directly from the database, or
+from a kiosk.
+
+A kiosk is a :doc:`dedicated device <hardware>` (a PC, tablet, or mobile phone) for employees to use
+when they check in and out.
+
+Kiosks are needed for employees who do **not** have access to the database.
+
+Only employees with access to the database can check in and out from the *Attendances* app, and they
+are referred to as *users*.
+
+.. important::
+   If employees :ref:`check in and out <attendances/kiosk-mode-entry>` using a badge or an RFID,
+   then an :doc:`accessible device <hardware>` in :ref:`*Kiosk Mode* <attendances/kiosk-mode>`
```

Comment:
Thanks for adding more context to this admonition, your refs are great!!
Just a quick fix to remove italic formatting :)
```suggestion
   then an :doc:`accessible device <hardware>` in :ref:`Kiosk Mode <attendances/kiosk-mode>`
```